ui/StatusQ/src/formatteddoubleproperty.cpp

```diff
@@ -0,0 +1,66 @@
+#include "StatusQ/formatteddoubleproperty.h"
+
+#include <QDebug>
+
+FormattedDoubleProperty::FormattedDoubleProperty(QObject* parent)
+    : QObject{parent}
+{
+    m_locale.setNumberOptions(QLocale::DefaultNumberOptions | QLocale::OmitGroupSeparator);
+}
+
+QVariant FormattedDoubleProperty::value() const
+{
+    return m_value;
+}
+
+void FormattedDoubleProperty::setValue(const QVariant& newValue)
+{
+    if (!newValue.isValid()) {
+        qWarning() << "Setting property to invalid value:" << newValue << "is not supported";
+        return;
+    }
+
+    auto ok = false;
+    auto tempValue = newValue.toDouble(&ok);
+
+    if (!ok && newValue.canConvert<QString>()) {
```

Comment:
Probably not a big deal but is it necessary to convert to string and back to double if `newValue` is already a double type?

Reply:
We are not converting _to_ string, but _from_ string and iff the conversion from a `double` fails